packages/pageheader/components/PageHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+
+import { BreadcrumbProps } from "../../breadcrumb/components/Breadcrumb";
+
+export interface PageHeaderProps {
+  children: {
+    title: React.ReactNode;
+    breadcrumb: React.ReactElement<BreadcrumbProps>;
+    actions?: React.ReactNode[];
+  };
+}
+
+class PageHeader extends React.PureComponent<PageHeaderProps, {}> {
+  public render() {
+    const { title, breadcrumb, actions = [] } = this.props.children;
+
+    return (
+      <div>
+        <h1>{title}</h1>
```

Comment:
I've never seen a page header in dcos-ui that shows both a page title _and_ breadcrumbs. Is the idea to visually hide the `h1` but leave it rendered in the DOM for accessibility?

If not, I think we just just have one prop (either `breadcrumb` or `title`) that renders something on the left

Reply:
Oh I guess I misunderstood the notes I made. It is not title + breadcrumb + actions but just breadcrumbs + actions as the title is part of the breadcrumb. Will change.

Reply:
Sorry @Poltergeist - that's my fault for misinterpreting how page headers work in DC/OS UI